.github/workflows/release-binaries.yml

```diff
@@ -148,6 +148,20 @@ jobs:
     if: github.repository == 'llvm/llvm-project'
     runs-on: ${{ inputs.runs-on }}
     steps:
+
+    - name: Checkout Actions
+      uses: actions/checkout@v4
+      with:
+        ref: ${{ (github.event_name == 'pull_request' && github.sha) || 'main' }}
+        sparse-checkout: |
+          .github/workflows/
+        sparse-checkout-cone-mode: false
+        path: workflows
+
+    - name: Setup Stage
```

Comment:
I'm assuming this moved up here so it can use the just pulled in action definition?

The subsequent LLVM checkout ends up overriding this?

Reply:
The next checkout won't override this, so I didn't have to move this.  The reason I did was to match the other stages which have these two steps next to each other.

Reply:
Makes sense.

For some reason I missed the `path` argument in my initial readover. This makes more sense now.

Reply:
I was wrong, the checkout does overwrite this.  See https://github.com/llvm/llvm-project/pull/100845.